src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
+
+## Observability
+
+Observability on the Giant Swarm platform is based in Mimir, Loki and Grafana. The stack runs in the management cluster and provides a centralized view of the platform metrics, logs, and traces. You can add application metrics and logs letting the your teams detect anomalies, troubleshoot issues, and optimize resources efficiently. The platform supports customizable dashboards and alerts, enabling proactive management and a deeper understanding of application behavior and infrastructure status.
+
+## Single sign-On
+
+Single Sign-On (SSO) simplifies user authentication across the Giant Swarm platform by allowing users to access multiple services with your existing identity provider. It enhances security and user convenience by reducing password fatigue and centralizing identity management. You can build on top of `Kubernetes` Role-Based Access Control (RBAC) to define fine-grained access policies and permissions, ensuring that only authorized users can interact with the platform resources.
+
+## Network and connectivity
+
+The different technologies used in the Giant Swarm platform provides you with a secure and reliable network and connectivity between your applications and services. Platform engineers can create network policies to control traffic flow and enforce security rules, ensuring that data is protected and isolated. Relying on the built-in networking capabilities of `Kubernetes`, the platform supports many routing and load balancing strategies, enabling seamless communication between services and applications.
+
+## Runtime security
+
+Runtime security focuses on protecting applications and infrastructure during execution. [Falco](https://www.cncf.io/projects/falco/) project provide you with intrusion detection, vulnerability scanning, and automated threat response capabilities. The Giant Swarm platform ensures that security policies are enforced consistently, reducing the attack surface and mitigating risks.
+
+## Cost management
+
+Cost control is becoming increasingly important as organizations seek to optimize their cloud spending. Based on lessons learned from our experience with customers, the platform is built to help you manage costs effectively. Our solution engineers will help you to add visibility and control over your resource usage and expenses, leading to better decision-making and cost savings.
```

Comment:
```suggestion
Cost control is becoming increasingly important as organizations seek to optimize their cloud spending. Based on lessons learned from our experience with customers, the platform is built to help you manage costs effectively. Our solution engineers will help you add visibility and control over your resource usage and expenses, leading to better decision-making and cost savings. By working on resource optimization and auto-scaling as well as resource projections and reservations, we can lower costs, without impacting every developer individually.
```

added the work we do in optimizing that goes beyond just visibility and control

Reply:
IMO still quite vague, or too long of a paragraph. Can we mention very explicitly what we do in this area? Auto-scaling and choice of instance types is one topic, but what else? Should we mention that we can do optimization projects with customers? Otherwise, it may sound like there's existing tooling to do all these things.